packages/core/src/common/classes.js

```diff
@@ -108,6 +108,9 @@ export const ORDERS = `${NS}-orders`
 // Book
 export const BOOK = `${NS}-book`
 
+// Chart
+export const CHART = `${NS}-chart`
+
```

Comment:
could you move this to /* page/containers - start...end */ section in this same file ?